### DISCUSSION TITLE
Landsat TM and MSS Collection 2 Data Request [Now available]

### DISCUSSION THREAD

**Discussion by cullen-molitor** [microsoft/PlanetaryComputer#22] [Data request]
I am a Masters of Environmental Data Science student at the Bren School of Environmental Science and Management. My capstone group project involves predicting crop yields with the [MOSAIKS](https://www.nature.com/articles/s41467-021-24638-z) machine learning approach. We have been granted access to the Planetary Computer and have been able to begin using the example [mosaiks.ipynb](https://github.com/microsoft/PlanetaryComputerExamples/blob/main/tutorials/mosaiks.ipynb) notebook. We would like to begin adapting this code to our use case with Landsat 5 data. 

I would like to request the Landsat [Landsat TM and MSS Collection 2](https://microsoft.github.io/AIforEarthDataSets/data/landsat-tm-mss.html) dataset. We were granted access to the data after following the directions in the above link. We have yet to use this method, but I think direct access from the data catalog would potentially streamline our workflow.

Thanks!

**Comment by TomAugspurger** [comment]
Hi Cullen,

> I think direct access from the data catalog would potentially streamline our workflow

Apologies, but I misunderstood your earlier message.  We're working on adding it this data to the [STAC catalog](https://planetarycomputer.microsoft.com/catalog/), in which case it will be accessible like any of the other datasets already there. But at the moment, that data is only available through Blob Storage.

**Reply by cullen-molitor** [reply]
I was working on the assumption that it would eventually be added. By adding a request here I was hoping to bump this collection up in the wish list que. My earlier message was mostly asking if that would be helpful or not and I can see how that was not very clear. Thanks for quick response!

**Comment by julietcohen** [comment]
Hello, I am also a master's student in data science, and I am interested in using either Landsat 5 or Landsat 7 data for my research. Satellite data from Landsat 5 or Landsat 7 are essentially my only options for my research question. I have already begun my project on the Planetary Computer, and I would really appreciate if the Landsat 5 or Landsat 7 data could be moved up on the que for the STAC catalog.

Do you have an estimate for how soon either of these datasets can be added to the STAC catalog? Would it be a matter of days or weeks?
Thank you!

**Reply by TomAugspurger** [reply]
Weeks / months is probably a better timeline for this being available through the STAC API.

**Comment by TomAugspurger** [comment]
Just noting that https://planetarycomputer.microsoft.com/dataset/landsat-c2-l1 has data from the MSS instrument on Landsat 1-5, and https://planetarycomputer.microsoft.com/dataset/landsat-c2-l2 has dataset from the `tm` instrument (and others) from Landsat 5-9.

**Comment by trchudley** [comment]
Hi @TomAugspurger,

Going to add to this discussion rather than open another one for now, but happy to open a new one if directed. I'm interested in using the 15m panchromatic band of Landsat 7-9 for feature identification, but they are only available in Collection 2 Level 1. Is there any possibility, or further interest, in adding at least the pan bands of L7-9 to the c2-l1 dataset?

Cheers,
Tom
